src/rollover.py

```diff
@@ -484,6 +485,54 @@ def main_ssh_test(args):
     return ssh.test(info[args.ec2_id]['PrivateIpAddress'])
 
 
+def main_check_for_task(args):
+    sys.stdout.write("Querying ECS ...")
+    sys.stdout.flush()
+
+    ecs_client = ecs.ECSClient(args.cluster)
+    ec2_client = ec2.EC2Client()
+    task_ids = ecs_client.list_tasks()
+    task_descriptions = ecs_client.describe_tasks(task_ids)
+    instance_map = {}
+    for instance in ecs_client.list_container_instances():
+        ecs_instance = ECSInstance(ecs_client, ec2_client, instance)
+        instance_map[ecs_instance.ecs_id] = ecs_instance
+
+    running_map = {}
+    for task in task_descriptions.values():
+        task_def = utils.pull_task_definition_name(task['taskDefinitionArn'])
+        ecs_id = utils.pull_instance_id(task['containerInstanceArn'])
+        running_map.setdefault(ecs_id, [])
+        if fnmatch.fnmatch(task_def, args.task_name_expr):
+            running_map[ecs_id].append(task_def)
```

Comment:
consider separating the functionality (492-507) into a helper method `get_matching_tasks_by_host` (returns a `running_map`). The printing ( <492, >509) can stay in main, or could also be in a method `print_running_map`

I realize there's not testing here (ideally we'd test `get_matching_tasks_by_host`), but would be nice to keep the testable unit separated.